tools/ktx/command.cpp

```diff
@@ -134,7 +131,11 @@ OutputStream::OutputStream(const std::string& filepath, Reporter& report) :
         #endif
         file = stdout;
     } else {
-        file = std::fopen(filepath.c_str(), "wb");
+#if defined(_WIN32)
+        file = _wfopen(DecodeUTF8Path(filepath).c_str(), L"wb");
```

Comment:
I'm surprised they don't have an overload of `fopen` that takes a `wstring`.

Reply:
There is none, as fopen comes from the C standard library, not from the C++ one. it's just that cstdio defines it in the std namespace. I'll actually change to use the "raw" fopen, as it's a bad C++ practice to use the c-prefixed headers for C standard library includes in C++.